helm/prometheus-rules/templates/alerting-rules/cert-manager.rules.yml

```diff
@@ -39,3 +39,15 @@ spec:
         severity: page
         team: cabbage
         topic: cert-manager
+    - alert: CertManagerTooManyCertificateRequests
+      annotations:
+        description: '{{`There are too many CertificateRequests in cluster {{ $labels.cluster_id }}.`}}'
+        opsrecipe: cert-requests-too-many/
+      expr: sum by (cluster_id) (etcd_kubernetes_resources_count{kind="certificaterequests.cert-manager.io"}) > 10000
```

Comment:
10000 is a completely arbitrary number. During the previous incident, there were ~75k CertificateRequests. Unfortunately we didn't have metrics to set a better number.

Now we are collecting metrics from most of the WC in viking (ref https://github.com/giantswarm/giantswarm/issues/22495) and we should have a better picture to set a new threshold in the future.

Reply:
That's fine. We have a lot of "arbitrary" number in our alerting that comes from experience.